x/bank/simulation/msgs.go

```diff
@@ -133,44 +111,32 @@ func sendAndVerifyMsgSend(app *baseapp.BaseApp, mapper auth.AccountKeeper, msg b
 	return nil
 }
 
-// SingleInputSendTx tests and runs a single msg multisend w/ auth, with one input and one output, where both
-// accounts already exist.
-func SingleInputMultiSendTx(mapper auth.AccountKeeper) simulation.Operation {
-	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simulation.Account, event func(string)) (action string, fOps []simulation.FutureOperation, err error) {
-		fromAcc, action, msg, abort := createSingleInputMsgMultiSend(r, ctx, accs, mapper)
-		if abort {
-			return action, nil, nil
-		}
-		err = sendAndVerifyMsgMultiSend(app, mapper, msg, ctx, []crypto.PrivKey{fromAcc.PrivKey}, nil)
-		if err != nil {
-			return "", nil, err
-		}
-		event("bank/sendAndVerifyTxMultiSend/ok")
-
-		return action, nil, nil
-	}
-}
-
 // SingleInputSendMsg tests and runs a single msg multisend, with one input and one output, where both
 // accounts already exist.
-func SingleInputMsgMultiSend(mapper auth.AccountKeeper, bk bank.Keeper) simulation.Operation {
+func SimulateSingleInputMsgMultiSend(mapper auth.AccountKeeper, bk bank.Keeper) simulation.Operation {
 	handler := bank.NewHandler(bk)
-	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simulation.Account, event func(string)) (action string, fOps []simulation.FutureOperation, err error) {
-		fromAcc, action, msg, abort := createSingleInputMsgMultiSend(r, ctx, accs, mapper)
-		if abort {
-			return action, nil, nil
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context, accs []simulation.Account, event func(string)) (
+		opMsg simulation.OperationMsg, fOps []simulation.FutureOperation, err error) {
+
+		fromAcc, comment, msg, ok := createSingleInputMsgMultiSend(r, ctx, accs, mapper)
+		opMsg = simulation.NewOperationMsg(msg, ok, comment)
+		if !ok {
+			return opMsg, nil, nil
 		}
 		err = sendAndVerifyMsgMultiSend(app, mapper, msg, ctx, []crypto.PrivKey{fromAcc.PrivKey}, handler)
 		if err != nil {
-			return "", nil, err
+			return opMsg, nil, err
```

Comment:
Ditto on logging an event also in the case of failure, so we can see how many simulated msgs are failing